features/step_definitions/cli_steps.rb

```diff
@@ -183,7 +183,7 @@ def modify_file(file_name, orig_text, new_text)
       line.strip.gsub('"', "'").gsub("'", '')
     end.join("\n")
 
-    file_content.should include(fragment.gsub("'", ''))
+    expect(file_content).to include(fragment.gsub("'", ''))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.